core/src/main/java/jenkins/security/seed/UserSeedChangeListener.java

```diff
@@ -32,14 +32,11 @@
 import java.util.logging.Logger;
 import jenkins.security.SecurityListener;
 import jenkins.util.Listeners;
-import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 /**
  * Listener notified when a user was requested to changed their seed
+ * @since TODO
```

Comment:
This will be awkward once the script runs and looks at when the line was changed.
```suggestion
 * @since 2.160 and 2.150.2, but restricted (unavailable to plugins) before TODO
```